### DISCUSSION TITLE
Plot Panel Custom Transformation/math modifier functions

### DISCUSSION THREAD

**Discussion by jplourenco** [foxglove/community#821] [Q&A]
Greetings, 

While doing some analysis I noticed that I required specific scaling functions for the being displayed in the Plot Panel that are not available in the default math modifiers. 

I was able to add the functions by changing adding int the source code.

However, Is there anyway to load additional math/transformation functions to the plot other than actually changing foxglove studio's source code?

**Comment by jtbandes** [comment]
Duplicate of https://github.com/orgs/foxglove/discussions/792
